src/combineReducers.js

```diff
@@ -74,6 +74,47 @@ function assertReducerSanity(reducers) {
   })
 }
 
+function getReducers(config) {
```

Comment:
This function doesn't really do anything reducer-specific. It is just grabbing those keys off of `config` whose values are functions. You could just as well call this `getFunctionValues`.